src/server/server_family.h

```diff
@@ -167,14 +156,15 @@ class ServerFamily {
   // if kDbAll is passed, burns all the databases to the ground.
   std::error_code Drakarys(Transaction* transaction, DbIndex db_ind);
 
-  LastSaveInfo GetLastSaveInfo() const;
+  detail::LastSaveInfo GetLastSaveInfo() const;
 
   // Load snapshot from file (.rdb file or summary.dfs file) and return
   // future with error_code.
   util::fb2::Future<GenericError> Load(const std::string& file_name);
 
   bool IsSaving() const {
-    return is_saving_.load(std::memory_order_relaxed);
+    std::lock_guard lk(save_mu_);
+    return save_controller_ && save_controller_->IsSaving();
```

Comment:
seems that this function is now used only for tests. 
I think that returning bool(save_controller_) is enough for this

Reply:
actually .. I debuged the test for a while to understand that I do need it, as it takes some time for the save executed command to be scheduled in the transaction queue. If I dont have this is_saving_ set after the scheduling the test SaveFlush fails as it will schedule flushall before the save

Reply:
Let's not complicate real code for testing reasons (especially without comments or TEST prefix), we have lot's of other ways to check if saving is in progress. For example check `tl_slice_snapshots`, which should be easiest. We can also check transaction metrics